modules/ztp-validating-the-generation-of-policy-crs.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-policy-crs_{context}"]
+
+= Validating the generation of policy CRs
+
+Policy custom resources (CRs) are generated in the same namespace as the PolicyGenTemplate from which they were created. The same troubleshooting flow applies to all policy CRs generated from PolicyGenTemplates regardless of whether they are common, group, or site based.
```

Comment:
Passive voice. Generated by what? Created by what?

Reply:
changed - "ArgoCD generates the policy custom resources (CRs) in the same namespace as the PolicyGenTemplate from which they were created."